inventory/all_projects/pas

```diff
@@ -3,3 +3,6 @@ slavery-prod1.princeton.edu
 [pas_staging]
 slavery-staging1.princeton.edu
 # slavery-staging2.princeton.edu
+[pas_staging_upgrade]
```

Comment:
We also need to add this new group to the `staging: children` list so that these VMs get the OS updates and other changes that run against the `staging` group.

Reply:
Good catch - thanks! Just made that change and committed.